File: Lecture03/Intents/app/src/main/java/com/codingblocks/intents/MainActivity.java
package com.codingblocks.intents;

import android.content.Context;
import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {
    EditText etVar1, etVar2;
    Button btnAdd;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        etVar1 = findViewById(R.id.etVar1);
        etVar2 = findViewById(R.id.etVar2);
        btnAdd = findViewById(R.id.btnAdd);

        btnAdd.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int a = Integer.valueOf(etVar1.getText().toString());
                int b = Integer.valueOf(etVar1.getText().toString());
                int c = a + b;

                Intent i = new Intent(MainActivity.this, ResultActivity.class);
                startActivity(i);
            }
        });

    }
}

File: Lecture14/AlarmManager/app/src/main/java/com/codingblocks/myapplication/MainActivity.kt
package com.codingblocks.myapplication

import android.app.AlarmManager
import android.app.PendingIntent
import android.content.Context
import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.os.SystemClock

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val i = Intent(this, MainActivity::class.java)
        i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
        i.addFlags(Intent.FLAG_ACTIVITY_MULTIPLE_TASK)
        val pi = PendingIntent.getActivity(
                this,
                123,
                i,
                PendingIntent.FLAG_ONE_SHOT
        )

        val am = getSystemService(Context.ALARM_SERVICE) as AlarmManager

        am.setRepeating(
                AlarmManager.ELAPSED_REALTIME,
                SystemClock.elapsedRealtime() + 60 * 1000,
                60 * 1000,
                pi
        )

    }
}
